test-infra/image-builder/roles/kubevirt-images/defaults/main.yml

```diff
@@ -124,3 +124,17 @@ images:
     checksum: sha256:289248945e2d058551c71c1bdcb31a361cefe7136c7fd88a09b524eedfaf5215
     converted: true
     tag: "latest"
+
+  openeuler-2203:
+    filename: openEuler-22.03-LTS-SP4-x86_64.qcow2.xz
+    url: https://mirrors.ocf.berkeley.edu/openeuler/openEuler-22.03-LTS-SP4/virtual_machine_img/x86_64/openEuler-22.03-LTS-SP4-x86_64.qcow2.xz
+    checksum: sha256:2dc9c9d73d172af0443d1a17786ee2e387006b9912fad24b5497ef103ecd7afb
+    converted: true
+    tag: "latest"
```

Comment:
Do we have a test for OpenEuler-2203?
I don't seem to see anything about OpenEuler-2203 testing?

Reply:
Yes, there is currently no CI (may not be needed), but the image has been pushed at: https://quay.io/repository/kubespray/vm-openeuler-2203